### PR TITLE
[front] - fix(vault): require one member group

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at requiring at least one member per vault, at creation and edition time.

**References:**
- https://github.com/dust-tt/tasks/issues/1321

## Risk

Low

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/vaults/CreateOrEditVaultModal.tsx [hunk 1]
```diff
@@ -107,15 +128,6 @@ export function CreateOrEditVaultModal({
           </DataTable.CellContent>
         ),
         enableSorting: false,
-        filterFn: (row: Row<RowData>, columnId: string, filterValue: any) => {
```
Comment (JulesBelveze) [inline front/components/vaults/CreateOrEditVaultModal.tsx:110]: Note: got rid of the `filterFn` on "name" as we now perform search through the `members/search` endpoint. The search is performed on emails, to not have to deal with diacritics along with sequelize...

File: front/components/vaults/CreateOrEditVaultModal.tsx [hunk 2]
```diff
@@ -97,6 +103,21 @@ export function CreateOrEditVaultModal({
   );
 
   const getTableColumns = useCallback(() => {
+    const manageMembers = (userId: string, addOrRemove: "add" | "remove") => {
```
Comment (Fraggle) [inline front/components/vaults/CreateOrEditVaultModal.tsx:106]: Can you move this into a hook ? (like useCreateVault etc.. ?)

File: front/components/vaults/CreateOrEditVaultModal.tsx (hunk 1 above)
Comment (Fraggle) [reply front/components/vaults/CreateOrEditVaultModal.tsx:110]: and names i hope ?

**Review by Fraggle — APPROVED** [review]

File: front/components/vaults/CreateOrEditVaultModal.tsx (hunk 1 above)
Comment (JulesBelveze) [reply front/components/vaults/CreateOrEditVaultModal.tsx:110]: @Fraggle for now the search is only performed on the email column for the above mentioned reasons. This decision was taken together with @flvndvd.

**Pull request merged**